src/envo/envo-edit.owl

```diff
@@ -44978,12 +44981,39 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605005> <http://purl.obolibrar
 # Class: <http://purl.obolibrary.org/obo/ENVO_03605006> (stream water)
 
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03605006> "Water which is part of a stream."@en)
-AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03605006> <https://orcid.org/ORCID:0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03605006> <https://orcid.org/0000-0001-9076-6066>)
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03605006> "2024-01-12T13:48:00Z"^^xsd:dateTime)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605006> "stream water"@en)
 EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_03605006> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_00002006> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_00000023>)))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605006> <http://purl.obolibrary.org/obo/ENVO_00002006>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605007> (freshwater stream)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03605007> "A stream which is composed primarily of fresh water.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03605007> <https://orcid.org/0000-0001-9076-6066>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03605007> "2024-01-10T19:48:00Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605007> "freshwater stream"@en)
+EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_03605007> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_00000023> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00002011>)))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605007> <http://purl.obolibrary.org/obo/ENVO_00000023>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605008> (freshwater stream biome)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03605008> "A freshwater biome which is determined by a freshwater stream.")
```

Comment:
Is this in pattern? Wouldn't freshwater stream biome be represented as a subclass of freshwater ecosystem, with the definition as templated in #1484 ?

Reply:
yeah I think you're right. I'll fix that tomorrow morning (ET)